tests/LaunchDarkly.ClientSdk.Tests/Internal/DataSources/StreamingDataSourceTest.cs

```diff
@@ -83,25 +83,25 @@ string expectedQuery
                     Assert.Equal(expectedQuery, req.Query);
                     Assert.Equal("GET", req.Method);
                 }
-            }
+            }
         }
 
         // REPORT mode is known to fail in Android (ch47341)
```

Comment:
Is this still true?

Reply:
I don't have any background knowledge on the claim.